blocks/library/paragraph/index.js

```diff
@@ -81,6 +84,39 @@ class ParagraphBlock extends Component {
 		this.nodeRef = node;
 	}
 
+	setFontSize( value ) {
+		const { setAttributes } = this.props;
+		if ( fontSizeValues.includes( value ) ) {
+			if ( fontSizeValues[ 0 ] === value ) {
+				setAttributes( { fontSize: fontSizeValues[ 0 ], textClass: 'is-small-text' } );
```

Comment:
In fact, there's redundancy between `fontSize` and `textClass`. I think we only need one or the other, probably the modifier "small", "regular", "large". Everything else (the class name, the pixel value) can be derived from the constant value.